html/elements/iframe.json

```
@@ -912,10 +912,10 @@
             "description": "<code>sandbox=\"allow-modals\"</code>",
             "support": {
               "chrome": {
-                "version_added": null
+                "version_added": "46"
               },
               "chrome_android": {
-                "version_added": null
+                "version_added": "46"
               },
               "edge": {
                 "version_added": null
```

This has probably reached some of the other Chrome-derived browsers.

```suggestion
                "version_added": "≤79"
```

And below this, outside of GitHub's suggestion range:

- Opera 33
- Opera for Android 33
- Samsung Internet 5.0

---

Thanks for the spot-on suggestions, I've added the respective versions.